Azure-Samples/communication-services-ios-calling-hero
# [Bug] Replace the speaker, video and microphone icons to 24px

## Purpose

Update the speaker, video and microphone icons 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
1. Navigate to the Lobby screen
2. Check video, microphone and speaker icons
3. Mute/unmute mic, enable/disable camera, select a speaker
4. Join or start a call
5. Repeat step 2
```

## What to Check
Verify that the following are valid
* Icons on the Lobby screen
* Icons on the Call screen for different orientations
